fennel/datasets/datasets.py

```diff
@@ -63,6 +63,7 @@
     get_python_type_from_pd,
     FENNEL_STRUCT_SRC_CODE,
     FENNEL_STRUCT_DEPENDENCIES_SRC_CODE,
+    WindowStruct,
```

Comment:
`WindowStruct` isn't nice at all. Let's just use `Window`?

Reply:
Yup this just a placeholder. I am thinking some names that we can use. Not in favour of Window though as then there will be three Window -> Window operator, Window ("5y") and then this Window Struct. Although we have deprecated the use of Window ("5y") and users won't be using Window operator proto nor class. So ideally we can do that.

Reply:
We had an internal discussion and the plan is 
1. Rename the Window("5y") to Last("5y"), since we will anyways be kicking it out its fine.
2. .window can create a WindowOperator ( this is internal struct name so doesn't matter much )
3. The externally visible struct will be called Window.

Let me know what you think ?

Reply:
I would say to do in this one, I dont expect the surface area to be large since we have anyways deprecated Window("") syntax, and would keep the number of windows limited